source/index.html.md

```diff
@@ -25,6 +25,7 @@ includes:
   - dataset_resources
   - dataset_metadata
   - dataset_attachments
+  - dataset_attachments
```

Comment:
It should be `dataset_alternative_exports` no?

Reply:
It sure should